addons/web/static/tests/views/kanban/kanban_view_tests.js

```diff
@@ -13483,7 +13483,50 @@ QUnit.module("Views", (hooks) => {
                         uid: 7,
                     });
                     assert.strictEqual(resModel, "partner");
-                    assert.deepEqual([...resIds], []);
+                    assert.deepEqual([...resIds], [1]);
+                },
+            });
+
+            const cpButtons = getVisibleButtons(target);
+            assert.deepEqual(
+                [...cpButtons].map((button) => button.textContent.trim()),
```

Comment:
```suggestion
                cpButtons.map((button) => button.textContent.trim()),
```
What am I missing? :thinking: